onnxruntime/core/providers/dml/DmlExecutionProvider/src/ExecutionProvider.cpp

```diff
@@ -38,7 +38,7 @@ namespace Dml
     using namespace onnxruntime::common;
```

Comment:
note: for viewing DML EP file changes, I suggest hiding whitespace diffs as my editor automatically removed trailing spaces

Reply:
Yeah, that seems to be a new VS Code behavior - unsure when it started doing that, but it happened to me too.

Reply:
I believe it would be the recent addition of .vscode\settings.json to the repo as that has

```json
    // Always remove trailing whitespaces
    "files.trimTrailingWhitespace": true,
```

Reply:
Diff tools generally have a way to ignore whitespace though. I use CodeFlow, but github has this:

<img width="267" alt="image" src="https://user-images.githubusercontent.com/979079/190525163-28984094-ac0f-4de2-8ac8-63a2b677f4ca.png">

Reply:
> ```json
> "files.trimTrailingWhitespace": true,
> ```

Thanks Scott - just set it to false locally. (and yes, CodeFlow rocks as an extension to GitHub - how I wish people outside MS could enjoy it too)